samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -30,6 +35,30 @@ void main(void)
 	if (tfm_status != TFM_SUCCESS) {
 		printk("tfm_ns_interface_init failed with status %d\n", tfm_status);
 	}
+
+	printk("Reading secure memory (FICR->INFO.PART)\n");
+	plt_err = tfm_platform_mem_read(&read_result,
+					(intptr_t)&NRF_FICR_S->INFO.PART, 4,
+					&err);
+	if (plt_err != TFM_PLATFORM_ERR_SUCCESS || err != 0) {
+		printk("tfm_platform_mem_read failed: plt_err: 0x%x, err: 0x%x\n",
+			plt_err, err);
+		return;
+	}
+	printk("FICR->INFO.PART: 0x%08x\n", read_result);
+
+	printk("Reading secure memory (FICR->INFO.VARIANT)\n");
+	plt_err = tfm_platform_mem_read(&read_result,
```

Comment:
Can we avoid duplicating the mem_read? I think it will be even more instructive to clearly see the difference between the calls.

Reply:
I don't understand what you want to change, could you elaborate?

Reply:
These 8 lines are almost-duplicated below and I was wondering if they could be put into a function or something instead

	plt_err = tfm_platform_mem_read(&read_result,
					(intptr_t)&NRF_FICR_S->INFO.PART, 4,
					&err);
	if (plt_err != TFM_PLATFORM_ERR_SUCCESS || err != 0) {
		printk("tfm_platform_mem_read failed: plt_err: 0x%x, err: 0x%x\n",
			plt_err, err);
		return;
	}
	printk("FICR->INFO.PART: 0x%08x\n", read_result);

Reply:
I see and agree. Check now.